pilot/pkg/features/pilot.go

```diff
@@ -571,6 +571,9 @@ var (
 	PrioritizedLeaderElection = env.RegisterBoolVar("PRIORITIZED_LEADER_ELECTION", true,
```

Comment:
can't comment on it directly - but go.mod and go.sum permission got changed. can you revert that?